pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -334,9 +335,7 @@ private static void start() throws MalformedURLException {
 
         before("*", new HttpHeaderMDCFilter(X_FORWARDED_FOR));
         before("*", new MDCLogBreadCrumbFilter());
-        before("*", (Request request, Response response) -> {
-            MDC.put(MDC_STUDY, RouteUtil.parseStudyGuid(request.pathInfo()));
-        });
+        before("*", (Request request, Response response) -> MDC.put(MDC_STUDY, RouteUtil.parseStudyGuid(request.pathInfo())));
```

Comment:
Not saying this is not a good idea. It is! But dropped in the middle of a 72 file PR that has a completely different purpose without a comment is not the best way to make this change clear and transparent.

Reply:
Agreed- this was a change in the PR prior to my work.

Reply:
It wasn't dropped, it was refactored. I just removed braces